### PR TITLE
Feature TMC#109 dekker use-case

### PR DESCRIPTION
**Pull request by Lurkerpas**

### CODE REVIEW

File: sdl2promela/builtins.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    return None
+
+
+def __translate_length(
+    call: sdlmodel.ProcedureCall, parameters: List[promelamodel.Statement]
+) -> promelamodel.Statement:
+    """
+    Length assigns an integer scalar, so there is no need to pack it into assign_value call
+    """
+    return (
```
Comment (rphh) [inline sdl2promela/builtins.py:31]: what about checking parameters?

Comment (Lurkerpas) [reply sdl2promela/builtins.py:31]: Fair point, will add checks

File: tests/integration/dekker/Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
+SPIN?=spin
```
Comment (rphh) [inline tests/integration/dekker/Makefile:1]: This shall be added to `tests/integration/Makefile`

Comment (Lurkerpas) [reply tests/integration/dekker/Makefile:1]: Added

**Pull request merged**